srcpkgs/zfs/patches/dkms.patch

```diff
@@ -1,44 +0,0 @@
-# Patch from https://aur.archlinux.org/packages/zfs-dkms/
-# Avoids recompiling ZFS userland utilities with DKMS rebuilds
-# Also short-circuits a useless test for Debian in the DKMS source
```

Comment:
Why did you remove this patch?

Reply:
By mistake; I meant to remove only the patch for linux5.6 that was made obsolete by upstream changes. The latest push restores this patch.